core/tests/revert-test/tests/tester.ts

```diff
@@ -10,7 +10,8 @@ export class Tester {
         public ethProvider: ethers.providers.Provider,
         public ethWallet: ethers.Wallet,
         public syncWallet: zkweb3.Wallet,
-        public web3Provider: zkweb3.Provider
+        public web3Provider: zkweb3.Provider,
+        public hyperchainAdmin: ethers.Wallet
```

Comment:
can you add a small comment on what is a hyperchain admin?

Reply:
Added, using the same wallet as ethWallet didn't work here, the server would fail after one of txs